### PR TITLE
Add dimple package

### PR DESCRIPTION
**Pull request by danielneal**
Add dimple package (issue #58)
Also fixes d3 - not sure if this needs to be a separate PR?

### CODE REVIEW

File: dimple/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# cljsjs/d3
```
Comment (martinklepsch) [inline dimple/README.md]: Please adjust to `cljsjs/dimple`

File: dimple/build.boot [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+       :description "A JavaScript charting library based on d3"
+       :url         "http://dimplejs.org/"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"MIT" "http://opensource.org/licenses/MIT"}})
+
+(deftask package []
+  (comp
+   (download :url "https://raw.githubusercontent.com/PMSI-AlignAlytics/dimple/master/dist/dimple.v2.1.3.js"
+             :checksum "F1189BD5961E043161930B5D2F38EA2A")
+   (download :url "https://raw.githubusercontent.com/PMSI-AlignAlytics/dimple/master/dist/dimple.v2.1.3.min.js"
```
Comment (martinklepsch) [inline dimple/build.boot]: Probably we should download from a specific revision here and not from `master`.

**Comment by martinklepsch** [comment]
Hey Daniel, sorry for taking so long to react to this. I added a few notes, let me know how you feel about those. Other than that I'm happy with it and look forward to merging it :)

**Comment by danielneal** [comment]
Hey Martin, 
Thanks! I've made those changes, let me know if there's anything else. 
Daniel

**Comment by martinklepsch** [comment]
@danielneal that was fast, will merge and push now :)

**Comment by martinklepsch** [comment]
Pushed `[cljsjs/dimple "2.1.2-0"]` & `[cljsjs/d3 "3.5.5-1"]`. :smiley:

**Comment by danielneal** [comment]
Great stuff! Thanks :)

**Pull request merged**